provider/k8s/k8s.go

```diff
@@ -258,9 +273,32 @@ func (p *Provider) initializeTemplates() error {
 		return errors.WithStack(err)
 	}
 
+	d, _ := p.Cluster.AppsV1().Deployments("cert-manager").Get("cert-manager", am.GetOptions{})
+
+	// println(d.Spec.Template.Labels["app.kubernetes.io/version"])
+
 	if p.CertManager {
-		if err := p.applySystemTemplate("cert-manager", nil); err != nil {
-			return errors.WithStack(err)
+		if d != nil && d.Spec.Template.Labels["app.kubernetes.io/version"] != "v1.9.0" {
+			println("updating cert-manager...")
+			p.deleteSystemTemplate("cert-manager", nil)
```

Comment:
i'm not checking for errors because there's a race condition between the deletion of the namespace and the deletion of the resources in the namespace.i.e delete command might return a "not found" error when deleting some resources bc it was already cascade deleted due to deleting the namespace.